Automattic/sensei
# Add/learner management enrol filter

Resolves #2977

#### Changes proposed in this Pull Request:

* Adds two additional filters in course learner management, 'Enrolled Learners' and 'Unenrolled Learners'
* The following properties of `Sensei_Learners_Main` have been changed from public to private:
```
$course_id
$lesson_id
$view
$page_slug
```

#### Testing instructions:

* Have a course with many learners enrolled and others having progress but not being enrolled.
* Go to Learner Management for the course and test that the new filters are working correctly. Also make sure that the previous ones didn't break.
* Ensure that the filters behave correctly when there are no learners.
* Test pagination.
